README.md

```diff
@@ -242,6 +242,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 * [go-colortext](https://github.com/daviddengcn/go-colortext) - Go library for color output in terminals.
 * [go-isatty](https://github.com/mattn/go-isatty) - isatty for golang.
 * [go-prompt](https://github.com/c-bata/go-prompt) - Library for building a powerful interactive prompt, inspired by [python-prompt-toolkit](https://github.com/jonathanslenders/python-prompt-toolkit).
+* [promptui](https://github.com/manifoldco/promptui) Interactive prompt for command-line applications https://www.manifold.co
```

Comment:
please remove the external link. also please fix your item line to be in form of
```
* [name](https://github.com/username/repo) – Description.
```

and put the link and alphabetical order.

Reply:
@kirillDanshin ok i fixed it.